content/azs.md

```diff
@@ -100,16 +100,16 @@ VMs total: 4
 
 ---
 
-## Adding and removing AZs do a deployment.
+## Adding and removing AZs to a deployment.
 
 > **_NOTE:_**
->   With some CPIs AZS are referenced in Subnet configs of the cloud-config networks block. This means that moving AZs can have networking implications. E.g. networks can be Zonal ( e.g. AWS Subnets are associated to a Zone) or Regional ( e.g. GCP Subnets are associated with a Region). This is important to keep in mind when planning an AZ migration.
+>   With some CPIs AZS are referenced in the Subnet configuration of the cloud-config networks block. This means that moving AZs can have networking implications. E.g. networks can be Zonal ( e.g. AWS Subnets are associated to a Zone) or Regional ( e.g. GCP Subnets are associated with a Region). This is important to keep in mind when planning an AZ migration.
```

Comment:
We can do a bit better, removing extra spaces next to parenthesis.

Plus, the “_referenced in the Subnet configuration of the cloud-config networks block_” can still be improved.

Something like “_With some CPIs, availability zones are tied to subnets, as specified by the networks section of the Cloud Config_”